kfs-core/src/main/java/edu/arizona/kfs/fp/batch/dataaccess/impl/CsvBankTransactionsValidatedFileType.java

```diff
@@ -0,0 +1,239 @@
+package edu.arizona.kfs.fp.batch.dataaccess.impl;
+
+import edu.arizona.kfs.sys.KFSConstants;
+import org.apache.commons.io.FilenameUtils;
+import org.apache.commons.lang.StringUtils;
+import org.apache.log4j.Logger;
+import org.kuali.kfs.gl.GeneralLedgerConstants;
+import org.kuali.kfs.sys.KFSKeyConstants;
+import org.kuali.kfs.sys.batch.BatchInputFileTypeBase;
+import org.kuali.kfs.sys.exception.ParseException;
+import org.kuali.rice.core.api.datetime.DateTimeService;
+
+import java.io.*;
+import java.util.List;
+
+/**
+ * CSV Bank Transaction Validated File that uses the Bank Parameters to determine order of columns and delimiter
+ * <p>
+ * Created by nataliac on 3/10/17.
+ */
+public class CsvBankTransactionsValidatedFileType extends BatchInputFileTypeBase {
+    private static final Logger LOG = Logger.getLogger(CsvBankTransactionsValidatedFileType.class);
+
+    protected String outputPath;
+    protected String loadFileName;
+
+    protected DateTimeService dateTimeService;
+
+
+    public void consolidateFiles( List<String> filesToProcess ) {
+        LOG.debug("Starting consolidating Bank Transaction Validated Files size="+filesToProcess.size());
+
+        // create and open the output file
+        File outputFile = getOutputFile(getOutputFilePath());
+
+        try (BufferedWriter outputBufferedWriter  =
+                         new BufferedWriter(new FileWriter(outputFile))) {
+
+            // for each input file, copy contents without the first line = batchTotal, to the consolidated output file
+            for (String filePath : filesToProcess) {
+                LOG.debug("Consolidating Bank Transaction Validated File="+filePath);
+                File file = openFile(filePath);
+                // remove the done file
+                deleteDoneFile(file);
+
+                try (BufferedReader inBufferedReader = new BufferedReader(new FileReader(file))) {
+                    boolean firstRecord = true;
+                    while (inBufferedReader.ready()) {
+                        // skip batch total on first line
+                        if (firstRecord) {
+                            String ignoredFirstRecord = inBufferedReader.readLine();
+                            firstRecord = false;
+                        } else {
+                            outputBufferedWriter.write(inBufferedReader.readLine() + "\n");
+                        }
+                    }
+                } catch (IOException e) {
+                        LOG.error("ERROR: Consolidate load transaction file: Failed to read/write at file "+filePath, e);
+                        throw new RuntimeException("ERROR: Consolidate load transaction file: Failed to read/write at file "+filePath, e);
+                } catch (Exception e) {
+                    LOG.error("ERROR: Failed to open the input file [" + filePath + "].", e);
+                    throw new RuntimeException("Failed to open the input file [" + filePath + "].", e);
+                }
```

Comment:
I'm trying to use a new Java7 syntax: try with closeable resources:
   try (BufferedReader inBufferedReader = new BufferedReader(new FileReader(file))) {....

that should automatically call close at the end of the block and save me writing a finally block.